src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/client/manageusers/ManageUsersApiClient.kt

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.client.manageusers
+
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import reactor.core.publisher.Mono
+
+@Component
+class ManageUsersApiClient(@Qualifier("manageUsersApiWebClient") private val manageUsersApiClient: WebClient) {
```

Comment:
Client to get the user details - display name for a username. Note that the endpoint only needs a valid token - no additional roles are needed. So no HAAR updates.